scripts/php.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+if brew list php; then
+  echo $'\n'
+  echo "Configuring PHP"
+  echo $'\n'
+
+  echo $'\n'
+  echo "Installing PECL extensions (xdebug) for each version of PHP"
+  echo $'\n'
+  brew unlink php@7.2
```

Comment:
Should just be able to `brew unlink php` here so it will always work with the latest version that was installed on this system

Reply:
Oh nice, I wasn't sure if `brew unlink php` would link older versions too.

Reply:
It should just link `php`, like any other package is always going to be the latest version of php that you installed